### PR TITLE
Add commonly used classes to static call ignore list for phpmd

### PR DESCRIPTION
**Pull request by jashish24**
https://github.com/axelerant/drupal-quality-checker/issues/1

### CODE REVIEW

File: phpmd.xml.dist [hunk 1]
```diff
@@ -25,7 +25,15 @@
           \DateTime,
           \DateInterval,
           \DateTimeZone,
-          \Drupal\Component\Utility\Html
+          \Drupal\Component\Utility\Html,
+          \Drupal\Component\Utility\UrlHelper,
```
Comment (jashish24) [inline phpmd.xml.dist:29]: Used commonly accessed utility classes but will find a way to add namespace (if possible) and raise another PR.

File: phpmd.xml.dist [hunk 2]
```diff
@@ -25,7 +25,15 @@
...
+          \Drupal\Core\Access\AccessResult,
+          \Drupal\Core\Url,
+          \Drupal\Core\Cache\Cache,
+          \Drupal\Core\Render\Element,
+          \Drupal\Core\Annotation\Action,
+          \Drupal\aggregator\Entity\Feed,
```
Comment (hussainweb) [inline phpmd.xml.dist:35]: @jashish24, are these classes used commonly? Can you give some examples?

Comment (jashish24) [reply phpmd.xml.dist:35]: `Element`, `Action` & `Feed`, not so much but I have seen use of `Cache` for `Cache::invalidateTags` a lot.

Comment (hussainweb) [reply phpmd.xml.dist:35]: Thanks. I'm specifically asking about the classes from `aggregator` module (Feed and Item).

Comment (jashish24) [reply phpmd.xml.dist:35]: I don't see those being used that often. I have just scanned through core, some contrib modules and took the ones that looked relevant.

Comment (hussainweb) [reply phpmd.xml.dist:35]: Okay, let's just keep the common ones here. We can't maintain a list of classes in each and every module. Projects can still change this list anyway, so we don't need to worry to make this list complete.

Comment (jashish24) [reply phpmd.xml.dist]: Removed `aggregator` classes.

**Comment by hussainweb** [comment]
I think this is a good start. Feel free to create a new PR for any classes that you discover.

**Pull request merged**